internal/util/kubernetes/object/set_test.go

```diff
@@ -45,32 +45,32 @@ func TestSet(t *testing.T) {
 
 	t.Log("verifying creation of an object set")
 	set := &Set{}
-	require.False(t, set.Has(ing1))
-	require.False(t, set.Has(ing2))
-	require.False(t, set.Has(ing3))
-	require.False(t, set.Has(tcp))
+	require.False(t, set.Get(ing1))
```

Comment:
These are not meant to pass now, right? Returning a string backed type won't (or shouldn't) "match" `true` or `false`.

Reply:
Yes, please ignore the obsolete tests.